src/main/coffeescript/view/OperationView.coffee

```diff
@@ -261,7 +261,7 @@ class OperationView extends Backbone.View
     response_body = pre
     $(".response_code", $(@el)).html "<pre>" + data.status + "</pre>"
     $(".response_body", $(@el)).html response_body
-    $(".response_headers", $(@el)).html "<pre>" + data.getAllResponseHeaders() + "</pre>"
+    $(".response_headers", $(@el)).html "<pre>" + data.getAllResponseHeaders().replace( /</g, '&lt;' ).replace( />/g, '&gt;' ) + "</pre>"
```

Comment:
escaping HTML is a little more than just replacing `<` and `>`. Please use `$('<div/>').text(data.getAllResponseHeaders()).html()` to get escaped HTML